src/gameengine.js

```diff
@@ -223,10 +228,38 @@ GameEngine.prototype = {
                 case 77: //M;
                     Howler.mute(that.muted = !that.muted);
                     break;
+                case 80: //P
+                    that.paused = !that.paused;
+                    break;
+                default:
+                    console.log(e.which);
             }
             //console.log(e.which);
             e.preventDefault();
         }, false);
+
+        this.ctx.canvas.addEventListener("blur", function() {
```

Comment:
We can leave a comment or something saying what this means, but unfortunately that is the name of the event.
https://developer.mozilla.org/en-US/docs/Web/Events/blur